dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -1102,6 +1341,7 @@ std::vector<RemoteRequest> DAGStorageInterpreter::buildRemoteRequests(const DM::
             filter_conditions,
             log));
     }
+    LOG_INFO(log, "remote request size: {}", remote_requests.size());
```

Comment:
```suggestion
    LOG_DEBUG(log, "remote request size: {}", remote_requests.size());
```
ref https://github.com/pingcap/tiflash/pull/7285, Shouldn't print too much info log